### PR TITLE
Add code from step 1 to step 7 in framework

### PR DESCRIPTION
**Pull request by RajKumar0696**
code for all the below cards
https://softwaretestinglearnings.atlassian.net/browse/FRM-2
https://softwaretestinglearnings.atlassian.net/browse/FRM-3
https://softwaretestinglearnings.atlassian.net/browse/FRM-4
https://softwaretestinglearnings.atlassian.net/browse/FRM-5
https://softwaretestinglearnings.atlassian.net/browse/FRM-6
https://softwaretestinglearnings.atlassian.net/browse/FRM-7

### CODE REVIEW

File: Utilies/CustomLogger.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    def loggen():
+        logging.basicConfig (filename=".\\Logs\\automation.logs",
+                             format='%(asctime)s: %(levelname)s: %(message)s',
+                            datefmt='%m/%d/%Y %I:%M:%S %p')
+
+                            
+        logger = logging.getLogger()
+        logger.setLevel(logging.INFO)
+        return logger
+C:\Users\WELCOME/.ssh/id_ed25519.pub
```
Comment (HemaSree1519) [inline Utilies/CustomLogger.py]: What is this last line 15 ? If this is a typo, please remove and push again

File: testCases/test_login.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+class Test_001_LogIn:
+    url = ReadConfig.get_application_url()
+    user_name = ReadConfig.get_user_name()
+    password = ReadConfig.get_password()
+    logger = LogGen.loggen()
+
+    def test_homePageTitle(self, setup):
+        self.logger.info("*****Test_001_LogIn*******")
+        self.logger.info("****** Home Page Title Checking *******")
```
Comment (HemaSree1519) [inline testCases/test_login.py:16]: Ideally there should not be any log/print statements present in the code pushed

File: Utilies/CustomLogger.py (hunk 1 above)
Comment (RajKumar0696) [reply Utilies/CustomLogger.py]: Thanks, removed the typo.

File: testCases/test_login.py (hunk 2 above)
Comment (RajKumar0696) [reply testCases/test_login.py:16]: This is meant to be logged into the log files, so it is needed.

**Review by HemaSree1519 — APPROVED** [review]

**Pull request merged**